extensions/proposals/WEBGL_video_texture/extension.xml

```diff
@@ -20,21 +20,21 @@
   <overview>
     <mirrors href="https://www.khronos.org/registry/gles/extensions/OES/OES_EGL_image_external.txt"
              name="OES_EGL_image_external">
-      <addendum>Defines a new texture target <code>TEXTURE_EXTERNAL_OES</code>.</addendum>
-      <addendum>Provides a mechanism for binding <code>HTMLVideoElement</code>'s EGLImage to external texture targets.</addendum>
-      <addendum>Provides time of frame, texture width and height of <code>HTMLVideoElement</code>'s EGLImage.</addendum>
+      <addendum>Defines a new texture target <code>TEXTURE_VIDEO_IMAGE</code>.</addendum>
```

Comment:
I think ultimately we need to specify on each platforms this new target is mapped to an user visible target, so developers know they need to rebind a previous texture to that target because bindTexture(TEXTURE_VIDEO_IMAGE) will destroy that binding. Could you add a TODO here to make sure we clarify this when we implement it on all platforms so we can finalize it?